docs/docs/03-syntax-and-usage/03-attributes.md

```diff
@@ -63,6 +63,30 @@ templ component() {
 <hr style="padding: 10px" class="itIsTrue" />
 ```
 
+## Spread attributes
+
+Using `{ spread... }` in an element. You can easily transfer attributes from a `map[string]string` into your elements.
+
+To simplify, you can use `templ.Attributes{}` instead of `map[string]string`.
+
+```templ
+templ component(attrs templ.Attributes) {
+  <p { attrs... }></p>
+}
+
+templ usage() {
+  @component(templ.Attributes{"data-testid": "paragraph"}) 
+}
+```
+
+```html title="Output"
+<p data-testid="paragraph">Text</p>
+```
+
+:::note
+You can use conditional attributes with the spread attributes.
```

Comment:
Rather than a note here, what about an example in the conditional attributes section that shows it in used?

Reply:
Something like this ?

![image](https://github.com/a-h/templ/assets/5224459/35d15462-e083-4543-afd8-61ee4dae9769)

Personally, I find that the example is a lot for a niche case.

Reply:
I don't think it needs a whole example in it's own. I thought you could just amend the example in https://templ.guide/syntax-and-usage/attributes#conditional-attributes to contain a spread?

Reply:
> I don't think it needs a whole example in it's own. I thought you could just amend the example in https://templ.guide/syntax-and-usage/attributes#conditional-attributes to contain a spread?

Like this ?

![image](https://github.com/a-h/templ/assets/5224459/2d5494ca-0d8d-43a1-a37d-a5e3f12e37ae)

I admit I find it cleaner